src/browser/base/content/ZenUIManager.mjs

```
@@ -112,8 +112,13 @@
 
   _updateOnHoverVerticalTabs() {
     let onHover = Services.prefs.getBoolPref('zen.view.sidebar-expanded.on-hover');
+    let expanded = this.expanded;
```

Biome

[error] This let declares a variable that is only assigned once.

'expanded' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)